extra/teamcollision.txt

```diff
@@ -13,7 +13,7 @@
 			"CGameMovement::PlayerSolidMask"
 			{
 				"linux"		"16"
-				"windows"	"17"
+				"windows"	"15" // 17 points to CGameMovement::CalcRoll
```

Comment:
Must leave out a misleading comment. Comment on issue usually goes into pull request.

Reply:
Ah sorry, I am still trying to get used to GitHub. I did a quick vtable scanning and found out that the current offset for that function was directing to another function.

Reply:
Yes, usually windows offsets's value go by -1 from linux, but I've missed this one out when I was correcting others @ 3eb88ece06eb2cf237b967a85a4eff97c0102ef1.